distributor-api-v1/deprecations/README.md

```diff
@@ -0,0 +1,7 @@
+# Deprecations
+
+This page contains a list of past and future deprecations in the API. Deprecation information are present in individual pages. `Deprecation notice` is a date from which we do support both versions (previous/deprecated and next/living standard). `Removal date` is a date where we aim to remove the previous/deprecated variant of the API.
+
+| Deprecation | Deprecation notice | Removal date |
+|:------------|:-------------------|:-------------|
+| [PaymentCardInput](payment-card-input.md) | 18.4.2021 | 31.7.2021 |
```

Comment:
In the connector, we have a similar table where we have another column with a replacement. Then you don't have to write the long paragraph you have below the table.
![image](https://user-images.githubusercontent.com/5834393/120279700-0d20f700-c2b7-11eb-84fb-9d171723513e.png)